apps/modernization-ui/src/apps/page-builder/page/management/edit/subsection/manage/ManageSubsectionTile/ManageSubsectionTile.tsx

```diff
@@ -31,104 +39,106 @@ export const ManageSubsectionTile = ({ subsection, setOnAction, action, onDelete
                                 <NbsIcon name="drag" size="3" />
                             </div>
-                            <div className={styles.warningGroup}>
+                            <div className={styles.label} data-testid={'label'}>
                                 <NbsIcon name="group" size="3" />
+                                <span data-testid="manageSectionTileId">{`${subsection.name} (${subsection.questions.length})`}</span>
                             </div>
-                            <div>{`${deleteWarning.name}(${subsection.questions.length})`}</div>
-                        </div>
-                        <div className={styles.warningModalBtns}>
-                            {isValidDelete(subsection) ? (
-                                <>
-                                    <div
-                                        onClick={() => {
-                                            onDelete?.(subsection);
-                                            setDeleteWarning(undefined);
-                                            setOnAction?.(false);
-                                        }}>
-                                        Yes, delete
-                                    </div>
-                                    <div className={styles.separator}>|</div>
-                                    <div
-                                        onClick={() => {
-                                            setDeleteWarning(undefined);
-                                            setOnAction?.(false);
-                                        }}>
-                                        Cancel
-                                    </div>
-                                </>
-                            ) : (
-                                <div
+
+                            <div className={styles.buttons}>
+                                <Button
+                                    type="button"
+                                    onClick={() => {
+                                        setOnAction?.(true);
+                                    }}
+                                    outline
+                                    disabled={action}
+                                    className={styles.iconBtn}>
+                                    <Icon.Edit style={{ cursor: 'pointer' }} size={3} />
+                                </Button>
+                                <Button
+                                    type="button"
+                                    className={styles.iconBtn}
+                                    outline
+                                    disabled={action}
                                     onClick={() => {
-                                        setDeleteWarning(undefined);
-                                        setOnAction?.(false);
+                                        setOnAction?.(true);
+                                        setDeleteWarning(subsection);
```

Comment:
Question: Does this break the delete functionality? Since it is setting action to true, not sure if the buttons will ever get enabled once a delete flow is ended?

Reply:
not sure why how this change snuck in, will revert